vrecord

```diff
@@ -37,6 +37,25 @@ MPVOPTS+=(--load-scripts=no)
 MPVOPTS+=(--script "${SCRIPTDIR}/qcview.lua")
 MPVOPTS+=(--really-quiet)
 
+# Set variables for system differences
+SYSTEM_TEST="$(uname -s)"
+if [[ "${SYSTEM_TEST}" = "Darwin" ]] ; then
+    OS_TYPE="macOS"
+    DEFAULTFONT='/Library/Fonts/Andale Mono.ttf'
```

Comment:
Suggest double quotes (here and below), also with this var assigned within the condition, then line 27 may be removed.